src/languageservice/services/yamlCompletion.ts

```diff
@@ -195,6 +195,16 @@ export class YAMLCompletion extends JSONCompletion {
             }
           }
         }
+        if (node.type === 'null') {
```

Comment:
Are you checking if there is a String "null" or if it actually `null`?

Reply:
I'm checking on string `'null'`

Reply:
ok